backend/pkg/rest/runtime_scan_controller.go

```diff
@@ -81,9 +83,12 @@ func (s *Server) GetRuntimeScanResults(params operations.GetRuntimeScanResultsPa
 	// in the case of no application IDs to look by, nothing was scanned, so we can just return
 	if len(state.runtimeScanApplicationIDs) == 0 {
 		return operations.NewGetRuntimeScanResultsOK().WithPayload(&models.RuntimeScanResults{
-			Counters:                 &models.RuntimeScanCounters{},
-			Failures:                 failures,
-			VulnerabilityPerSeverity: []*models.VulnerabilityCount{},
+			CisDockerBenchmarkCountPerLevel: []*models.CISDockerBenchmarkLevelCount{},
+			CisDockerBenchmarkCounters:      &models.CISDockerBenchmarkScanCounters{},
+			CisDockerBenchmarkScanEnabled:   quickScanConfig.CisDockerBenchmarkScanEnabled,
```

Comment:
it's not correct to show whether CIS benchmark was enabled in the last runtime scan by fetching it from config, because it can be set post-scan.